src/main/java/com/parkhomovsky/bookstore/service/OrderService.java

```diff
@@ -0,0 +1,26 @@
+package com.parkhomovsky.bookstore.service;
+
+import com.parkhomovsky.bookstore.dto.order.OrderDto;
+import com.parkhomovsky.bookstore.dto.order.OrderPlaceRequestDto;
+import com.parkhomovsky.bookstore.dto.order.OrderUpdateStatusRequest;
+import com.parkhomovsky.bookstore.dto.order.StatusUpdateResponseDto;
+import com.parkhomovsky.bookstore.dto.orderitem.OrderItemDto;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.core.Authentication;
+
+public interface OrderService {
+    OrderDto process(OrderPlaceRequestDto orderPlaceRequestDto,
+                     Authentication authentication);
+
+    StatusUpdateResponseDto updateStatus(Long orderId,
+                                         OrderUpdateStatusRequest updateStatusRequest);
+
+    List<OrderItemDto> getOrderItemsDto(Pageable pageable, Long orderId,
```

Comment:
```suggestion
    List<OrderItemDto> getOrderItems(Pageable pageable, Long orderId,
```

it's not necessary to specify Dto in method name